src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -215,6 +218,39 @@ protected void Initialize(StorageEnvironment environment, DocumentDatabase docum
             }
         }
 
+        private void LoadValues()
+        {
+            TransactionOperationContext context;
+            using (_contextPool.AllocateOperationContext(out context))
+            using (var tx = context.OpenWriteTransaction())
```

Comment:
Why have two separate transactions here? One in CreateSchema and one here?
We can use just a single transaction